### PR TITLE
new package build workflow

### PR DESCRIPTION
**Pull request by hopkiw**

### CODE REVIEW

**Comment by hopkiw** [comment]
currently live as `liamh-testing-package-build-2`

File: concourse/tasks/generate-package-version.yaml [hunk 1]
```diff
@@ -4,19 +4,29 @@ platform: linux
...
 inputs:
-- name: token
+- name: repo
 
 outputs:
 - name: package-version
 
 run:
-  path: sh
+  path: ash
```
Comment (gaohannk) [inline concourse/tasks/generate-package-version.yaml:16]: sh?

Comment (hopkiw) [reply concourse/tasks/generate-package-version.yaml:16]: no ash is the shell provided in alpine

**Pull request merged**